infra/helper.py

```diff
@@ -721,9 +721,14 @@ def docker_run(run_args, print_output=True, architecture='x86_64'):
   """Calls `docker run`."""
   platform = 'linux/arm64' if architecture == 'aarch64' else 'linux/amd64'
   command = [
-      'docker', 'run', '--rm', '--privileged', '--shm-size=2g', '--platform',
+      'docker', 'run', '--privileged', '--shm-size=2g', '--platform',
       platform
   ]
+  if not os.getenv('OSS_FUZZ_SAVE_CONTAINERS_NAME'):
+      command.append('--rm')
+  else:
+      command.append('--name')
+      command.append(os.getenv('OSS_FUZZ_SAVE_CONTAINERS_NAME'))
```

Comment:
nit: flip the logic for simplicity.
```python
if os.getenv('OSS_FUZZ_SAVE_CONTAINERS_NAME'):
  ...
else:
  ...
```

Reply:
Sounds good, done

Reply:
Thanks, let's wait for @oliverchang to have a final look.